packages/radix-ui-themes/src/styles/reset.css

```diff
@@ -1,4 +1,4 @@
-.radix-themes[data-is-root-theme='true'] {
+.radix-themes:where([data-is-root-theme='true']) {
```

Comment:
`[data-is-root-theme='true']`?

Reply:
Same comment really regarding the attribute name…